### PR TITLE
Feature/log normal

### PR DESCRIPTION
**Pull request by rnugent3**

### CODE REVIEW

File: Statistics/Distributions/LogNormal.cs [hunk 1]
```diff
@@ -68,18 +74,19 @@ private void addRules()
         #region Functions
         #region IDistribution
         public override double PDF(double x){
-            Normal sn = new Normal();
-            return sn.PDF(Math.Log(x));
+            Normal standardNormal = new Normal();
```
Comment (HenryGeorgist) [inline Statistics/Distributions/LogNormal.cs:80]: we should use the Normal.StandardNormal(p) instead

File: Statistics/Distributions/LogNormal.cs [hunk 2]
```diff
@@ -35,13 +35,33 @@ public LogNormal()
         /// <param name="mean"></param> mean of the logged data
         /// <param name="sd"></param> standard deviation of the logged data
         /// <param name="sampleSize"></param>
-        public LogNormal(double mean, double sd, int sampleSize = int.MaxValue)
+        ///  <param name="unlogged"></param> Unlogged is true if the mean and standard deviation reflect unlogged data 
+        public LogNormal(double mean, double sd, int sampleSize = int.MaxValue, bool unlogged = false)
         {
-            Mean = mean;
-            StandardDeviation = sd;
+            if (unlogged == false)
```
Comment (HenryGeorgist) [inline Statistics/Distributions/LogNormal.cs]: do we really need this feature? will it ever be used by our code in fda?

File: Statistics/Distributions/LogNormal.cs [hunk 3]
```diff
@@ -35,32 +35,15 @@ public LogNormal()
         /// <param name="mean"></param> mean of the logged data
         /// <param name="sd"></param> standard deviation of the logged data
         /// <param name="sampleSize"></param>
-        ///  <param name="unlogged"></param> Unlogged is true if the mean and standard deviation reflect unlogged data 
-        public LogNormal(double mean, double sd, int sampleSize = int.MaxValue, bool unlogged = false)
+        public LogNormal(double mean, double sd, int sampleSize = int.MaxValue)
```
Comment (HenryGeorgist) [inline Statistics/Distributions/LogNormal.cs:38]: good.

**Review by HenryGeorgist — APPROVED** [review]
looks good to me. I suggested one small change for efficency.

File: Statistics/Distributions/LogNormal.cs (hunk 1 above)
Comment (rnugent3) [reply Statistics/Distributions/LogNormal.cs:80]: all I see in the normal class is `public static double StandardNormalInverseCDF(double p)` I do not see a Normal.StandardNormal(p)

File: Statistics/Distributions/LogNormal.cs (hunk 2 above)
Comment (rnugent3) [reply Statistics/Distributions/LogNormal.cs]: I was attempting to make better use of Table 6-3 in EM 1110-2-1619 according to the footnote in that table, but was unsuccessful, you'll see I later deleted that line.

**Pull request merged**